### DISCUSSION TITLE
Can I include own SFC's in the docs

### DISCUSSION THREAD

**Discussion by vanvanni** [vuepress/core#368] [Q&A] [answered]
Hi,

I want to to use: https://v2.vuepress.vuejs.org/guide/markdown.html#components
But include my own vue components to document my css framework. Is this possible?

**Comment by Mister-Hope** [comment, marked as answer]
You can use them directly in markdown files after you global register them in `clientAppEnhanceFile`.

https://vuejs.org/api/application.html#app-component
